### DISCUSSION TITLE
Hi i need some help, so i dont know how to run this and i need help, can someone help?

### DISCUSSION THREAD

**Discussion by MythISsntDead** [hxr404/Discord-Console-hacks#44] [Q&A] [answered]

**Comment by Raz-js** [comment, marked as answer]
Goolge: Enable Dev Tools In Discord And Check The Reddit Link

**Comment by hxr404** [comment]
@MythISsntDead Discord disabled the console in the desktop client, but as @Raz-js mentioned a quick google search will show you how to re-enable it.

The console hacks should continue to work in browsers, though. To open the Console in Firefox, simply press Ctrl + Shift + I.

**Comment by RobinRMC** [comment]
This thread should be moved to the Q&A category, and then marked as Answered.
